nerdlets/shared/container-panel.js

```diff
@@ -92,13 +92,13 @@ export default class ContainerPanel extends React.Component {
   async load() {
     this.setState({});
     const { account, containerId } = this.props;
-    const accountId = account.id;
+    const accountIds = account.id;
```

Comment:
This appears to just be a single account id, do we need to update the name of this variable?

Reply:
will fix.